### ISSUE TITLE
z loss hot fix

### ISSUE THREAD

**Issue by lunaB** [sju-coml/2020-Font-picker#17]
```
# [gen.en] z vector 유사도 로스
t_z = encoder(t)[0]
z_loss = mse_criterion(z, t_z)
```

를

```
t_z = encoder(d_out)[0]
z_loss = mse_criterion(z, t_z)
```
로 수정해야함 왜냐하면 마음이 아프기 때문
저위치에 타겟을 넣는다는것은 폰트와 메타정보를 매치시킨다는것에서 의미가있지만 학습적인 측면에서 크게 도움이 되지않음.
그렇기에 위와같은 변화가 필요함
